output/output.cpp

```diff
@@ -70,6 +70,9 @@ void Output::outputBuffer(void *mem, size_t size, int64_t timestamp_us, uint32_t
 
 Output *Output::Create(VideoOptions const *options)
 {
+	if (options->codec == "libav")
+		return new Output(options);
+
```

Comment:
Hehe. An output that does nothing at all. I love it!

Reply:
If for some reason a user wanted to output a .pts file when using libav, this would allow it. Not sure how useful that may be, given the timestamps are also encoded in the container...